src/app/modules/dialogs/weekly-profit-dialog/weekly-profit-dialog.component.ts

```diff
@@ -32,6 +108,76 @@ export class WeeklyProfitDialogComponent extends ChartGeneralMethodsComponent im
         ['Weekly Profit K$', 'right', '#0085ff'],
         ['All profit M$', '1', '#eeb000']
       ]);
+      this.loadSecondChart(data);
+
     });
   }
+
+  getPlatformFromVaultName(vaultName: string): string {
+      if (vaultName.startsWith('V_')) {
+          vaultName = vaultName.replace('V_', '');
+      }
+      const underscoreCount = vaultName.split('_').length;
+      let platform = 'SINGLE';
+      if (underscoreCount > 0) {
+          platform = vaultName.split('_')[0];
+      } else if (vaultName.indexOf('CRV') >= 0) {
+          platform = 'CRV';
+      }
+      return platform;
+  }
+
+  getCurrentDate(item): string {
+      const date = new Date(item.blockDate * 1000);
+      const currentDate = [date.getFullYear(), date.getMonth() + 1, date.getDate()].join('/');
+      return currentDate;
+  }
+
+  loadSecondChart(data): void {
+      const times: Array<string> = data.map(item => this.getCurrentDate(item));
+
+      this.option.xAxis[0].data = times;
+      const groups = data.reduce((acc, item) => {
+          const vaultName = item.vault;
+          const platform = this.getPlatformFromVaultName(vaultName);
+          if (!acc[platform]) {
+              acc[platform] = [];
+          }
+          acc[platform].push(item);
+          return acc;
+      }, {});
+
+      let seriesCounter = 0;
+      this.option.legend.data = [];
+      this.option.series = [];
+      for (const key in groups) {
+          if (groups.hasOwnProperty(key)) {
+              this.option.series.push({
+                  name: key,
+                  type: 'line',
+                  stack: '总量',
+                  areaStyle: {},
+                  emphasis: {
+                      focus: 'series'
+                  },
+                  data: []
+              });
+              const newDataList = [];
+
+              for (const i of times) {
+                  let value = 0;
+                  for (const j of groups[key]) {
+                      const currentDate = this.getCurrentDate(j);
+                      if (i === currentDate) {
+                          value = j.weeklyAllProfit / 1000;
```

Comment:
let's use fullRewardUsdTotal cumulative values
fullRewardUsdTotal - it is a cumulative sum of progits for 1 vault in USD